pkg/scaling/scale_handler.go

```diff
@@ -611,3 +607,171 @@ func (h *scaleHandler) getScaledObjectState(ctx context.Context, scaledObject *k
 
 	return isScaledObjectActive, isScalerError, metricsRecord, nil
 }
+
+type scalerMetrics struct {
+	queueLength float64
+	maxValue    float64
+	isActive    bool
+}
+
+// / --------------------------------------------------------------------------- ///
+// / ----------             ScaledJob related methods               --------- ///
+// / --------------------------------------------------------------------------- ///
+
+// getScaledJobMetrics returns metrics for specified metric name for a ScaledJob identified by its name and namespace.
+// It could either query the metric value directly from the scaler or from a cache, that's being stored for the scaler.
+func (h *scaleHandler) getScaledJobMetrics(ctx context.Context, scaledJob *kedav1alpha1.ScaledJob) []scalerMetrics {
+	// TODO this loop should be probably done similar way the ScaledObject loop is done
```

Comment:
could you please address this TODO?

Reply:
Hi @zroubalik 
Cause I don't know the context very well, could you elaborate what todo means? BTW, I've changed it like below. I guess it's not similar any more.
I changed it like the below one, but I haven't deleted the line.

before
https://github.com/kedacore/keda/pull/4707/files#diff-e15b5649231cd2fc16ca66df8b66cc1c705c0daf7007b23a9a017bf0948409c4L239

now
https://github.com/kedacore/keda/pull/4707/files#diff-7e9a23e26ddcd00c13026a9c8f9bedcb7414c4cf1fe36a3a8ff1f29258fec055R632

Reply:
Yeah, then please remove the TODO

Reply:
Done.